<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetComputerDialog</class>
 <widget class="QDialog" name="SetComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>604</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="11" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Количество SSD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDAmount"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>ИД блока питания</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputPSUId"/>
      </item>
     </layout>
    </item>
    <item row="18" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>350</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonSetComputer">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCloseDialog">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ИД материнской платы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputMotherboardId"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>ИД корпуса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCaseId"/>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Название корпуса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCaseName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ИД процессора</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCPUId">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_33">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Количество SSD M.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDM2Amount"/>
      </item>
     </layout>
    </item>
    <item row="14" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_40">
      <item>
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>ИД вентилятора</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFanId"/>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Количество HDD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputHDDAmount"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ИД оперативной памяти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputRAMId"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ИД видеокарты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputVideocardId"/>
      </item>
     </layout>
    </item>
    <item row="17" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_47">
      <item>
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>ИД кулера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCoolerId"/>
      </item>
     </layout>
    </item>
    <item row="14" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_39">
      <item>
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>Название вентилятора</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFanName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>ИД HDD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputHDDId"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Количество оперативной памяти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputRAMAmount"/>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>ИД SSD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDId"/>
      </item>
     </layout>
    </item>
    <item row="17" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_48">
      <item>
       <widget class="QLabel" name="label_46">
        <property name="text">
         <string>Название кулера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCoolerName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_26">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Название HDD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputHDDName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Название процессора</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCPUName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Название SSD M.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDM2Name">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="16" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_42">
      <item>
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>Название СЖО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputWCSName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Название блока питания</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputPSUName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_32">
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Название SSD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="15" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_38">
      <item>
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>Количество вентиляторов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFanAmount"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Название видеокарты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputVideocardName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_34">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>ИД SSD M.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputSSDM2Id"/>
      </item>
     </layout>
    </item>
    <item row="16" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_41">
      <item>
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>ИД СЖО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputWCSId"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Название материнской платы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputMotherboardName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Название оперативной памяти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputRAMName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>ИД сборки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputId">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SetComputerDialog</sender>
   <signal>finished(int)</signal>
   <receiver>SetComputerDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>SetComputerDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
